tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -480,6 +575,17 @@ INSTANTIATE_TEST_SUITE_P(
 INSTANTIATE_TEST_SUITE_P(
     ReductionAxisIsOne,
     ATenNodesParametrizedTest,
-    testing::Values(std::make_tuple(Sizes({m, 1}), Sizes({1, n}))));
+    testing::Combine(
```

Comment:
Off-topic: It's not needed here, but for new tests I wonder if we could combine these kinds of parametrizations? For example since we use k=32, we could parametrize the test with a boolean flag `k_is_one` and if the flag is true we would replace any 32s in the shapes with 1 before running the test. That would reduce the need to repeat all the inputs for each case, but it makes it harder to filter to only the concrete K tests since the tests are simply numbered. It would be great if the parameters were reflected in the name like they are in pytest...

Reply:
We could add another flag, and replace the value of K within tests. I am not sure if there is a way to replace the values before they are passed to a test: that is, the test directly views K=1.  I'll look into it.